diesel_compile_tests/tests/fail/derive_deprecated/deprecated_belongs_to.stderr

```diff
@@ -0,0 +1,138 @@
+warning: #[belongs_to] attribute form is deprecated
```

Comment:
Not really. I was just printing the warning as soon as I see it. And since I was testing bad formats on the old deprecated format too, you would see the warning first and then the error.

Your change fixed that because we now only warn if the format is okay. So, we don't see this warning anymore.